ui/index.html

```diff
@@ -1,13 +1,28 @@
 <!DOCTYPE html>
```

Comment:
this is simply to get unit tests/etc to pass as our build expects this file to be here. it will get overwritten during the `mage` or `mage dev` tasks and is added to .gitignore so any changes will not be committed to git